cookbooks/retention.md

```diff
@@ -0,0 +1,211 @@
+*Authored by the Product Data Science Team. Please direct questions/concerns to Ben Miroglio (bmiroglio).*
```

Comment:
drive-by: you'll have to add `Miroglio` to the `.spelling` file (similar with `interpretability`). You can either also put `bmiroglio` in the dictionary or surround it with backticks.

Reply:
Thanks for the guidance--Fixed!